todoapp/src/components/DeletePopUP.vue

```diff
@@ -0,0 +1,168 @@
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue'
```

Comment:
You don't need to import defineProps and defineEmits are built in macros which can be access without importing them
You may also receive a warning in the terminal when you start the app saying the same, please make sure you don't have any error or warning in terminal before creating the PR